b2share/modules/schemas/root_schemas/root_schema_v0.json

```diff
@@ -5,132 +5,191 @@
-            "resource_type": {
+            "language": {
+                "title": "Language",
+                "description": "The name of the language the document is written in.",
+                "type": "string",
+                "default": "en"
+            },
+            "resource_types": {
                 "title": "Resource Type",
-                "description": "The type of the resource.",
+                "description": "The type(s) of the resource.",
                 "type": "array",
                 "items": {
-                    "type": "string",
-                    "enum": ["Audiovisual", "Collection", "Dataset", "Event", "Image",
-                        "InteractiveResource", "Model", "PhysicalObject", "Service",
-                        "Software", "Sound", "Text", "Workflow", "Other"]
+                    "type": "object",
+                    "properties": {
+                        "resource_type": {
+                            "type": "string"
+                        },
+                        "resource_type_general": {
+                            "enum": ["Audiovisual", "Collection", "Dataset", "Event", "Image", "InteractiveResource", "Model", "PhysicalObject", "Service", "Software", "Sound", "Text", "Workflow", "Other"]
+                        }
+                    }
                 },
                 "uniqueItems": true
             },
-            "alternate_identifier": {
-                "title": "Alternate identifier.",
+            "alternate_identifiers": {
+                "title": "Alternate identifiers",
                 "description": "Any kind of other reference such as a URN, URI or an ISBN number.",
-                "type": "string"
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "alternate_identifier": {
+                            "type": "string"
+                        },
+                        "alternate_identifier_type": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "uniqueItems": true
+            },
+            "descriptions": {
+                "title": "Descriptions",
+                "description": "A more elaborate description of the resource. Focus on a content description that makes it easy for others to find, and to interpret its relevance.",
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "description": {
+                            "type": "string"
+                        },
+                        "description_type": {
+                            "enum": ["Abstract", "Methods", "SeriesInformation", "TableOfContents", "TechnicalInfo", "Other"]
+                        }
+                    }
+                },
+                "uniqueItems": true
```

Comment:
If we follow OpenAire guidelines "descriptions" must contain at least the "Abstract". But in our case it was optional up to now.

In case you want to do it, JSON Schema [does not have yet a "contains" constraint](https://github.com/json-schema/json-schema/wiki/contains-(v5-proposal)). However we could force the first item in the list to be the Abstract:
```json
{
  "title": "Descriptions",
  "description": "A more elaborate description of the resource. Focus on a content description that makes it easy for others to find, and to interpret its relevance.",
  "type": "array",
  "items": [
    {
      "type": "object",
      "properties": {
        "description": {
          "type": "string"
        },
        "description_type": {
          "enum": [
            "Abstract"
          ]
        }
      },
      "required": [
        "description_type",
        "description"
      ]
    }
  ],
  "additionalItems": {
    "type": "object",
    "properties": {
      "description": {
        "type": "string"
      },
      "description_type": {
        "enum": [
          "Abstract",
          "Methods",
          "SeriesInformation",
          "TableOfContents",
          "TechnicalInfo",
          "Other"
        ]
      }
    },
    "required": [
      "description_type",
      "description"
    ]
  },
  "minItems": 1
}
```